docs/configuration.rst

```diff
@@ -140,6 +140,10 @@ Mongodb
 - ``MONGODB_PORT`` (default: ``27017``)
 - ``MONGODB_USERNAME`` (default: ``""``)
 - ``MONGODB_PASSWORD`` (default: ``""``)
+- ``MONGODB_REPLICA_SET`` (default: ``""``)
```

Comment:
Why isn't this variable taken into account in the forum config?

Reply:
In the forum config, a URI with `mongodb+srv://...` is used and in this case there is no need to specify the replica set. The replica set is needed only when the shard hosts are explicitly listed in the `MONGODB_HOST`. I have tested and this works for the forum

Reply:
Then if a replica set mongodb is used, do both MONGODB_HOST and MONGODB_REPLICA_SET need to be configured?

Reply:
Then is it possible for forum to use replica set config if MONGODB_REPLICA_SET is configured, while MONGODB_PROTOCOL is configured as "mongodb"?

Reply:
@zakum1 Could you give an example of how you define all these variables to connect to Atlas? It would really help us better understand the use case. We are trying hard to minimize the number of Tutor config variables, which is why we ask so many questions :innocent:

Reply:
OK, I think I understand: the `replicaSet` parameter is only required because the LMS/CMS connection string is outdated and does not make use of the `MONGODB_PROTOCOL`. Am I right? If yes, I think we can assume that the mongodb library will be updated soon-ish (by us or someone else) to support `MONGODB_PROTOCOL`. This will make the `replicaSet` parameter obsolete. If, for the moment, the `MONGODB_REPLICA_SET` configuration parameter can be replaced by a simple Tutor plugin, I'd rather document it than add a soon-deprecated configuration variable.

Am I making sense? If my interpretation is incorrect I think we should have a call to clarify things.


Could you please add a paragraph to the configuration docs (in the [mongodb section](https://docs.tutor.overhang.io/configuration.html#mongodb)) explaining how to connect to a MongoDB cluster?

Reply:
What I was proposing in the comment above was:

- not adding the `MONGODB_REPLICA_SET` config variable.
- documenting how to write a plugin that would allow users to connect to an Atlas cluster.

I think we need to talk about this :) Can you please send me an email?